index.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
+require('rootpath')()
```

Comment:
It's not the normal convention, so will hinder ease of contribution. I think we should be verbose in this case.